tools/mapping-scaling-tester/create_plots.py

```diff
@@ -25,39 +38,26 @@ def parseArguments(args):
         default="result",
         help="The prefix for all generated PDF plots.",
     )
+    parser.add_argument(
+        "-m",
+        "--mode",
+        default="mapping",
+        choices=["weak_scale", "mapping"],
```

Comment:
```suggestion
        choices=["weak_scaling", "mapping"],
```
?
How/ do these scripts work for a strong scaling?